cluster-scope/base/operators.coreos.com/operatorgroups/openshift-operators-redhat/operatorgroup.yaml

```diff
@@ -1,6 +1,5 @@
-apiVersion: operators.coreos.com/v1
```

Comment:
Was removal of apiversion here intentional?

Reply:
It was not!